pkg/proxy/ipvs/proxier.go

```diff
@@ -273,6 +273,12 @@ type Proxier struct {
 	// Inject for test purpose.
 	networkInterfacer     utilproxy.NetworkInterfacer
 	gracefuldeleteManager *GracefulTerminationManager
+	// localNoEndpoints represents the number of rules that couldn't be applied due to
+	// the absence of local endpoints.
```

Comment:
a comment reference the metric this is used for would be helpful, same for clusterNoEndpoints

Reply:
Done for both.